index.html

```diff
@@ -1,214 +1,255 @@
+            </h2>
+        </section>
+        <section class="text">
+            <p>
+                I can help you build a product , feature or website Look through some of my work and <br>experience! If you like what you see and have a project you need coded,  <br>don’t hesitate to contact me.
+            </p>
+        </section>
+        <button>
+            Get my resume
+        </button>
+    </div>
+    <div class="me-container">
+        <section class="card">
+            <section id="lang">
+            </section>
+            <h2>Languages</h2>
+            <ul>
+                <li>JavaScript </li>
+                <li>Ruby </li>
+                <li>HTML </li>
+                <li>CSS </li>
+            </ul>
+        </section>
+        <section class="card">
+            <section id="fw">
+            </section>
+            <h2>Frameworks</h2>
+            <ul>
+                <li>React.js</li>
+                <li>Ruby on Rails </li>
+                <li>RSpec </li>
+                <li>Capybara </li>
+                <li>Selenium </li>
+            </ul>
+        </section>
+        <section class="card">
+            <section id="sk">
+            </section>
+            <h2>Skills</h2>
+            <ul>
+                <li>Database Management </li>
+                <li>Version Control </li>
+                <li>CLI </li>
+                <li>Web Development </li>
+                <li>API Design </li>
+            </ul>
+        </section>
+    </div>
+    <div class="form-container">
+        <section class="touch">
+            <h2> 
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </h2>
+        </section>
+        <form class="connect" action="https://formspree.io/f/mwkjdnnq" method="POST" autocomplete="off">
+            <input id="name" name="username" placeholder="Full name" required type="text" maxlength="30"/>
+            <input id="mail" name="mail" placeholder="Email adress" required type="email"/>
+            <textarea id="comment" name="comment" maxlength="500" placeholder="Hey I’d like to connect to... " required></textarea>
+            <button>
+                Get in touch
+            </button> 
+        </form>  
+        <section class="mail">
+            <p>ngachaian@gmail.com</p>
+        </section>        
+     </div> 
+     <footer class="foot">
+        <section class="socials">
+            <ul>
+                <li> <a href=""> <img src="Images/twitter.png" alt="Twitter"/> </a> </li>
+                <li> <a href=""> <img src="Images/Linkedin icon.png" alt="LinkedIn"/> </a> <li>
+                <li> <a href=""> <img src="Images/medium.png" alt="Medium"/> </a> </li>
+                <li> <a href=""> <img src="Images/github.png" alt="GitHub"/> </a> </li>
+                <li> <a href=""> <img src="Images/angellist.png" alt="Angellist"/> </a> </li>
+            </ul>
+        </section>
+    </footer>
```

Comment:
Your project is looking nice gradually. However, as suggested by the previous reviewer,  kindly make sure that your project matches that of the figma design to prove your level of professionalism and standard. You need to take your time, and follow all the dimensions of each elements in the figma design in order to match it.

- [ ] The widths, height, margins, paddings and other dimensions of the left hand text as well as the contact form must match the actual design as shown below. 

- [ ] I can see you added `max-width` here. Remember, applying a max-width means that the element cannot expand above the stated number. This can also limit you elements from being responsive. Instead of using the max-width, you can consider using percentage (%) or viewport width (vw) to make it more responsive. Kndly check [this link](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Responsive_Design) for further details on how to make it responsive. :+1: 

### Your design 
![Screenshot from 2023-06-17 12-09-22](https://github.com/NgachaIan/Personal_Portfolio/assets/65233860/6f027fe9-88f2-4ca0-9825-043bb011ea10)


### Actual design 

![Screenshot from 2023-06-17 12-12-00](https://github.com/NgachaIan/Personal_Portfolio/assets/65233860/5144a860-c5ed-40bf-84e9-a3806c3b10e6)